.changeset/popover.md

```diff
@@ -0,0 +1,69 @@
+---
+'@leafygreen-ui/popover': major
+---
+
+[LG-4445](https://jira.mongodb.org/browse/LG-4445): Replaces `usePortal` prop with `renderMode` prop with values of `'inline'`, `'portal'`, and `'top-layer'`. `renderMode="inline"` and `renderMode="portal"` are deprecated, and all popover elements should migrate to using the top layer. The old default was `usePortal = true`, and the new default is `renderMode = 'top-layer'`.
+  - When `renderMode="top-layer"` or `renderMode` is `undefined`, the popover element will render in the top layer using the [popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API)
+    - Adds `dismissMode` prop to control dismissal behavior of the popover element. [Read more about the popover attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/popover)
+    - Adds `onToggle` prop to run a callback function when the visibility of a popover element rendered in the top layer is toggled
```

Comment:
Unrelated to this PR, but do we provide examples of how to use `dismissMode` and `onToggle`?